### DISCUSSION TITLE
[UI/UX] Add red span color to the "required" label *

### DISCUSSION THREAD

**Discussion by dstrukt** [btcpayserver/btcpayserver#2368] [General]
I suggest we make the * at the end of a required field red. This is a very minor change, but I think it helps make the "required" fields a bit more scannable.

Before
<img width="1349" alt="Screen Shot 2021-03-14 at 4 02 19 PM" src="https://user-images.githubusercontent.com/6250771/111165764-a532fe00-855c-11eb-92bd-b12d7403e32c.png">

After
<img width="657" alt="Screen Shot 2021-03-14 at 3 58 51 PM" src="https://user-images.githubusercontent.com/6250771/111165817-b419b080-855c-11eb-930f-d77c5480986b.png">

**Comment by pavlenex** [comment]
Concept ACK.

**Comment by bolatovumar** [comment]
Made a PR here: https://github.com/btcpayserver/btcpayserver/pull/2373
